com.civicrm.hrjobroles/js/src/job-roles/controllers/hr-job-roles-controller.js

```diff
@@ -19,53 +19,53 @@ define([
       var fundersContacts = {};
       var roles_type = ['funders', 'cost_centers'];
 
-      $scope.format = HR_settings.DATE_FORMAT;
-      $scope.loading = true;
-      $scope.past_job_roles = [];
-      $scope.present_job_roles = [];
-      $scope.collapsedRows = []; // Tracks collapsed / expanded rows
-      $scope.contactList = []; // Contact List IDs array to use for the select lists
-      $scope.edit_data = {}; // Tracks edit data changes on the forms
-      $scope.view_tab = []; // Tracks clicked tabs per each row
-      $scope.CalendarShow = []; // As default hide the datepickers
-
+      vm.contactId = $scope.$parent.contactId;
```

Comment:
why is $scope used here, is there another way to get contactId? maybe it could be defined as app constant?